main/src/com/google/refine/expr/functions/strings/Reinterpret.java

```diff
@@ -66,7 +66,7 @@ public Object call(Properties bindings, Object[] args) {
                 return reinterpret(str, decoder, encoder);
             }
         }
-        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + " expects 2 or 3 arguments");
+        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + " expects String to reinterpret with a given target encoder and optional source decoder");
```

Comment:
Hmm, perhaps we should make this more user friendly... since the [original enhancement on 2.5 says](https://github.com/OpenRefine/OpenRefine/wiki/Changes-for-2.5):
> reinterpret() now supports optionally specifying a source encoding to be used instead of the project encoding - reinterpret("target encoding", "source encoding")

Maybe
```suggestion
        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + " expects String to reinterpret with a given target encoding and optional source encoding");

Reply:
Yes, definitely "encoding" not "encoder," or, more specifically, "character encoding"